tests/operator/operator.go

```diff
@@ -848,6 +858,11 @@ var _ = Describe("[sig-operator]Operator", Serial, decorators.SigOperator, func(
 					patch.WithReplace("/spec/imageRegistry", curRegistry),
 				)
 
+				// Re-enable the decentralized live migration feature gate
+				kv.Spec.Configuration.DeveloperConfiguration.FeatureGates = append(kv.Spec.Configuration.DeveloperConfiguration.FeatureGates, featuregate.DecentralizedLiveMigration)
+				patches.AddOption(
+					patch.WithReplace("/spec/configuration/developerConfiguration/featureGates", kv.Spec.Configuration.DeveloperConfiguration.FeatureGates),
+				)
```

Comment:
Ideally this would be in the PR that enables the feature gate by default, but I guess it doesn't really matter.

Reply:
I can take it out, doesn't matter that much.